dashboard/lib/tasks/seed.rake

```diff
@@ -185,8 +185,11 @@ namespace :seed do
     script_files = opts[:script_files] || SCRIPTS_GLOB
     begin
       custom_scripts = script_files.select {|script| File.mtime(script) > scripts_seeded_mtime}
-      _, custom_i18n = Script.setup(custom_scripts, show_progress: Rake.application.options.trace)
-      Script.merge_and_write_i18n(custom_i18n)
```

Comment:
I think it's safe to ignore i18n here, because the only time any i18n was getting generated was when ScriptDSL found some in the .script file here: https://github.com/code-dot-org/code-dot-org/blob/28cc92d20cb5ef14547153327a1d006c80c19e0f/dashboard/app/models/script.rb#L1047-L1049

So, the change in this PR preserves existing behavior.

this begs the question, **should seeding migrated units update i18n?** I think the answer is no, because seeding doesn't seem like the right time to update i18n. I would expect that only to be necessary when a script is updated on levelbuilder via LessonsController#update. 

a practical argument for letting this behavior go away is that this takes us closer to the levelbuilder machine owning scripts.en.yml, which would reduce the chance of merge conflicts on scripts.en.yml. the old code allowed every environment to merge i18n into scripts.en.yml during seeding, which has resulted in failed deploys not just in staging but also test and production.

If we're comfortable with this path, we should inform other engineers and let them know to use levelbuilder mode locally if they want to edit units locally from now on. then the unit update codepath will take care of updating scripts.en.yml. engineers will need to learn how to do this anyway for all but the simplest of string changes, such as when they need to add a new lesson or level to /s/allthethings .

Reply:
oops, it looks like we are getting some i18n from get_lesson_groups_i18n here: https://github.com/code-dot-org/code-dot-org/blob/staging-next/dashboard/app/models/script.rb#L1381 I think I will need to rework this section of code.

Reply:
ah, just kidding, that is on the unit update path, which happens only on levelbuider. so I think my original reasoning is still accurate, and we shouldn't need to update i18n on this codepath.

Reply:
What you have laid out here makes sense to me. I'm a little nervous wondering if somehow scripts.en.yml could get out of sync with the information in script_json files by some accident (such as forgetting to keep the scripts.en.yml changes around locally as you update scripts).

Reply:
when you talk about updating scripts locally, do you specifically mean using the levelbuilder UI on localhost to make edits? in that case, we should still get updates to scripts.en.yml, here: https://github.com/code-dot-org/code-dot-org/blob/5a2529a51f8775ff839319f1b1a0cea248a64a0a/dashboard/app/models/script.rb#L1348

Reply:
I think that does make me less concerned.